src/prefect/cli/server.py

```diff
@@ -400,6 +414,32 @@ def ascii_welcome(ui_port="8080"):
     return message
 
 
+@server.command(hidden=True)
+def stop():
+    """
+    This command stops all Prefect Server containers that are connected to the
+    `prefect-server` network. Note: This will completely remove the `prefect-server` network
+    and all associated containers.
+    """
+    import docker
+
+    client = docker.APIClient()
+
+    network_id = client.networks(names=["prefect-server"])[0].get("Id")
+
+    if not network_id:
+        click.echo("No running Prefect Server found")
+        return
+
+    click.echo("Stopping Prefect Server containers and network")
+    for container in client.inspect_network(network_id).get("Containers").items():
```

Comment:
Is this a `dict.items` call? If so, it looks like you could use `keys()` instead of indexing out `container[0]` below.